main.js

```diff
@@ -1,31 +1,104 @@
 let api_url = 'https://jsonplaceholder.typicode.com/users';
-fetch(api_url)
-.then(function(response){
-    return response.json();
-})
-.then(function(information){
+let myArray=[];
+console.log(myArray);
+let data;
+let result;
+function getData(){
+    return fetch(api_url)
+    .then((response) => {
+        myArray = response.json();
+        return myArray;
+    }); 
+}
+console.log(myArray)
```

Comment:
remove console logs

Reply:
done and there is another console.log inside the searchtable function I removed it